Specs/DateTime/Korean/DateExtractor.json

```diff
@@ -508,33 +469,30 @@
   },
   {
     "Input": "나는 2개월 전에 돌아갔어. ",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "2개월 전에",
+        "Text": "2개월 전",
```

Comment:
Why shouldn't "에" be part of the entity?

Reply:
"에" should mean "on" and in other languages it is not extracted in most cases.

Reply:
The potential issue here is that not including it would require tokenization. Is there any side effect of including it?
I'd assume the "전에" here has a single meaning.

Reply:
I see, the problem is "에" can be replaced by other suffixes. For example "2개월 전부터" (from 2 months ago), "2개월 전후로" (around 2 months ago)... So including them will cause problems in other entities.

Reply:
But these other cases are tagged in English. So they should also be covered in Korean.

Reply:
yes, but my point is: in a pattern like "2개월 전부터", "2개월 전" should still be extracted as Date and the modifier "부터" only added in a later stage. We cannot extract the whole "2개월 전부터" directly as Date because it represents a Period. But if we completely exclude this pattern from Date, then we will fail to extract a Period patterns like "from Date1 to Date2" which relies on the DateExtractor to find Date1 and Date2 (e.g. "2016년 1월 12일**부터** 2016년 1월 22일**까지**").

Reply:
You are correct. Let's make sure these cases are extracted correctly with the modifiers.
Thanks for the detailed responses.